### PR TITLE
Np 46861 allow app admin to search for all tickets

### PR DESCRIPTION
**Pull request by truhacevkir**
It solves following tasks:
https://sikt.atlassian.net/browse/NP-46863
https://sikt.atlassian.net/browse/NP-46861
https://sikt.atlassian.net/browse/NP-46860

It makes possible to see count of minted doi's, count of doi's requested and count of requested general support cases.

### CODE REVIEW

File: search-commons/src/main/java/no/unit/nva/search/ticket/TicketParameter.java [hunk 1]
```diff
@@ -73,6 +73,7 @@
  */
 public enum TicketParameter implements ParameterKey {
     INVALID(ParameterKind.INVALID),
+    STATISTICS(IGNORED),
```
Comment (truhacevkir) [inline search-commons/src/main/java/no/unit/nva/search/ticket/TicketParameter.java:76]: not sure this is a good name for query param, any suggestions?

File: search-commons/src/test/java/no/unit/nva/search/ResourceClientTest.java
Comment (StigNorland) [inline search-commons/src/test/java/no/unit/nva/search/ResourceClientTest.java:1]: please add a test to resource_test_urls.json and ticket_datasource_urls.json

File: search-commons/src/main/java/no/unit/nva/search/ticket/TicketParameter.java (hunk 1 above)
Comment (StigNorland) [reply search-commons/src/main/java/no/unit/nva/search/ticket/TicketParameter.java:76]: not at the moment

File: search-commons/src/test/java/no/unit/nva/search/ResourceClientTest.java
Comment (StigNorland) [inline search-commons/src/test/java/no/unit/nva/search/ResourceClientTest.java:1]: please add a test to resource_test_urls.json and ticket_datasource_urls.json

**Review by StigNorland — CHANGES_REQUESTED** [review]

**Review by StigNorland — APPROVED** [review]
Supert

**Pull request merged**